python/ray/train/callbacks/logging.py

```diff
@@ -129,6 +129,12 @@ def handle_result(self, results: List[Dict], **info):
             f.seek(0)
             json.dump(loaded_results + [results], f, cls=SafeFallbackEncoder)
 
+    def finish_training(self, error: bool = False, **info):
+        logger.info(
+            "The training results are saved to %s"
+            % self.logdir
```

Comment:
Can we log the full filename instead of the logdir?

Reply:
Sure, replaced with `log_path`.

Reply:
Sure, replaced with `log_path`.